README.md

```diff
@@ -11,6 +11,8 @@ urlFragment: microsoft-teams-apps-quickresponses
 | [Documentation](https://github.com/OfficeDev/microsoft-teams-apps-quickresponses/wiki/Home) | [Deployment guide](https://github.com/OfficeDev/microsoft-teams-apps-quickresponses/wiki/Deployment-Guide) | [Architecture](https://github.com/OfficeDev/microsoft-teams-apps-quickresponses/wiki/Solution-Overview) |
 | ---- | ---- | ---- |
 
+## Quick Responses app

```

Comment:
Make this #
And change it to "Quick Responses App Template"
(same for the other repos)

Reply:
done, made the same change in other repos.